spinnman/transceiver.py

```diff
@@ -2344,23 +2393,35 @@ def malloc_sdram(self, x, y, size, app_id, tag=None):
     def free_sdram(self, x, y, base_address, app_id):
         """ Free allocated SDRAM
 
+        This method is currently deprecated and untested as there is no
+        known use. Same functionaility provided by ybug and bmpc.
+        Retained in case needed for hardware debugging.
```

Comment:
I don't think ybug does this actually.  This *might* be a case for full delete though, as we don't normally free SDRAM from host...

Reply:
marked likely to be removed